sql/chunk.sql

```diff
@@ -35,3 +35,13 @@ CREATE OR REPLACE FUNCTION _timescaledb_internal.calculate_chunk_interval(
         dimension_coord BIGINT,
         chunk_target_size BIGINT
 ) RETURNS BIGINT AS '@MODULE_PATHNAME@', 'ts_calculate_chunk_interval' LANGUAGE C;
+
+-- Function for explicit chunk exclusion. Supply relation/alias and an array of chunk ids as input.
```

Comment:
Misleading comment. Does not take relation/alias.

I.e., you cannot do:

```
SELECT chunks_in('my_hypertable', { 1, 3, 4 });
```

For a hypertable `my_hypertable` that exists in the current schema.

Reply:
What do you suggest to name it?

Reply:
Sure. I thought that relation/alias is more user friendly. Replacing it with `row`